joomla/joomla-cms
# [5.1] Use local $container instead of getting from Factory

Pull Request for Issue # .

### Summary of Changes
Just a small code improvement to System - Schema.org plugin. We can use the container from local variable `$container` to get database service instead of calling `Factory::getContainer()` to get the container.

### Testing Instructions
1. Code review should be enough
2. For human testing :
- Use Joomla 5.1
- Add or edit an article, select one **Schema Type** for that article in **Schema** tab and enter data, save it and make sure no error happens.


### Actual result BEFORE applying this Pull Request
Works !


### Expected result AFTER applying this Pull Request
Works !


### Link to documentations
Please select:
